README.md

```diff
@@ -46,7 +46,9 @@ Awesome Remote Work
 - [https://www.wfh.io/](https://www.wfh.io/), 100% remote tech-oriented job board
 - [http://jobs.remotive.io/](http://jobs.remotive.io/), weekly update with remote jobs from startups.
 - [http://remoteok.io/](http://remoteok.io/) a remote job aggregator. Job from Angel List, Stackoverflow, ...
-- [http://remotus.com/](http://remotus.com/) a fulltime jobs for developers, still under construction. Seems have a lots of startup jobs.
+- [http://remotus.com/](http://remotus.com/) a fulltime jobs for developers, still under construction. Seems have a lots of
+- [http://www.candoerz.com/](http://www.candoerz.com/) hourly update remote, relocate, freelance jobs aggregator. 
+- startup jobs.
```

Comment:
@adnanaman Could you please edit and resend PR, or I will add candoerz.com directly to the list? Thanks for contributing, mate!